platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/item/message/WorkflowItemMessageRowsLoader.java

```diff
@@ -26,16 +29,41 @@ public WorkflowItemMessageRowsLoader(AGWorkflowItem item) {
 
 	public List<WorkflowItemMessageRow> loadAllRows() {
```

Comment:
I think we need `WorkflowItemMessageRowsLoaderTest`.

Reply:
that will take 1h or 2h, so, will not do today